python/ml4ir/model/architectures/dnn.py

```diff
@@ -2,15 +2,22 @@
 from tensorflow.keras import layers
 
 
-def get_architecture_op(num_nodes=512):
+def get_architecture_op(num_nodes=256):
```

Comment:
as discussed elsewhere, this method could take a dict of "num_nodes" -> int, "dropout_1" -> float, "dense_2_dim" -> int, "final_dropout" -> float, which could carry around the defaults from some higher place (eventually externalized in a config).

Reply:
I want to do a **very** simple config sooner rather than later. Let me take a shot.